### ISSUE TITLE
unintialized constant Cream::Role::User

### ISSUE THREAD

**Issue by coder2000** [kristianmandrup/cream#4]
I try running any of the generators I get:

```
/home/******/.rvm/gems/ruby-head/gems/cream-0.7.0/lib/cream/role.rb:12:in `available': uninitialized constant Cream::Role::User (NameError)
```

**Comment by kristianmandrup** [comment]
Please use the latest version of cream, version 0.7.3. I am also planning to release some major updates this week, so stay tuned.

**Comment by coder2000** [comment]
Trying to install 0.7.3 but it keeps getting hung up on sugar-high dependencies:

```
Bundler could not find compatible versions for gem "sugar-high":
  In Gemfile:
    cream (~> 0.7.3) depends on
      sugar-high (~> 0.2.10)

    cream (~> 0.7.3) depends on
      sugar-high (0.3.0)
```

**Comment by kristianmandrup** [comment]
There should be NO sugar-high 0.2.x dependencies for any gem. If you find one or more, please report them :) Maybe I have not released all my latest gems yet. I have had problems with Internet access in the past weeks. Sorry.
To fix it for now, just clone the gems and update dependencies manually (to 0.3.0 fx) and install.

**Comment by coder2000** [comment]
I have changed my gemfile to use the git repo and it still gives me the error. I think the cream gem in rubygems is confusing it.

**Comment by kristianmandrup** [comment]
In the Gemfile, try setting one of the gem options for the gem statement in such as :path or :github

gem 'cancan-permits', '~> 0.3.0', :path => '/Users/kristianconsult/Development/Languages/Ruby/Apps/Gems/cancan-permits'

Uses your local gems repo version. Added advantage that it makes it much easier to debug as any changes in that local dir is auto-reflected when running the Rails app ;)

**Comment by kristianmandrup** [comment]
Fixed in Cream 0.8 and above
